tailwind.config.js

```diff
@@ -2,6 +2,13 @@ module.exports = {
   purge: [],
   darkMode: false, // or 'media' or 'class'
   theme: {
+    colors: {
+      foregreground: {
+        base: '#1E2433',
+        dark: '#101217'
+      },
+      white: '#fff'
```

Comment:
exista deja aceasta culoare, te rog sa o scoti din config. `bg-white` sau `text-white`. Also, culorile te rog sa le scoti (de la inia 5, tot obiectul) si sa le muti in extend. altfel ti le sterge pe toate care exista by default in tailwind si ramai doar cu ale tale. presupun ca de asta ai crezut ca nu mai exista alb :))